packages/guideline-blocks-settings/src/helpers/customCoordinatesGetterFactory.ts

```diff
@@ -13,21 +13,25 @@ export const customCoordinatesGetterFactory =
             const height = activeNode?.offsetHeight ?? 0;
 
             switch (event.code) {
```

Comment:
```suggestion
            switch (event.code as KeyboardCode) {
```

So we could remove the eslint-disable